src/plugins/pronoundb/index.ts

```diff
@@ -81,6 +84,60 @@ export default definePlugin({
             default: true
         }
     },
+
+    commands: [
+        {
+            description: "Set local pronouns for a person",
+            name: "setpronouns",
+            options: [
+                {
+                    description: "The user for which you want to set the local pronoun override",
+                    name: "user",
+                    displayName: "User",
+                    type: ApplicationCommandOptionType.USER,
+                    required: true,
+                },
+                {
+                    name: "pronouns",
+                    displayName: "Pronouns",
+                    type: ApplicationCommandOptionType.STRING,
+                    description: "What pronouns should I display for this user.",
+                    required: false,
+                    choices: [
+                        ...Object.keys(PronounMapping).map(id => ({
+                            label: PronounMapping[id],
+                            value: id,
+                            name: PronounMapping[id],
+                        })),
+                        {
+                            label: "Clear local override",
+                            name: "Clear local override",
+                            value: "clear",
+                        }
+                    ],
+                }
+            ],
+            inputType: ApplicationCommandInputType.BOT,
```

Comment:
```suggestion
            inputType: ApplicationCommandInputType.BOT,
            type: ApplicationCommandType.USER
```

basidally just this and then it would show up in context menu > apps
but I remember trying this once and it not working. The fix might or might be trivial, I haven't bothered looking into it

Reply:
This still does not work in all contexts (for example it does not work in servers), and i could not get the arguments to work to figure out what to change the pronouns to.
![image](https://user-images.githubusercontent.com/20768569/213445365-5fb2e097-a65c-47ca-9aa9-55712ad9e1ec.png)